src/blog/understanding-ecmascript-part-1.md

```diff
@@ -117,29 +123,42 @@ If an algorithm throws an exception, it means returning a Completion Record with
 
 [`ReturnIfAbrupt(argument)`](https://tc39.es/ecma262/#sec-returnifabrupt) means taking the following steps:
 
+:::ecmascript-algorithm
+<!-- markdownlint-disable blanks-around-lists -->
```

Comment:
We only need these when `:::ecmascript-algorithm` contains nothing but a list.

Reply:
Is there any way to configure markdownlint to ignore all these instances?

Reply:
Not that I know of. I don't think we want it for all of them though.

Reply:
Okay, let's go with this for now, we can always iterate later.